tests/_posix_spawn.py

```
@@ -0,0 +1,82 @@
+can't deadlock on a full ~64 KB OS buffer.
+"""
+
+from __future__ import annotations
+
+import os
+import select
+import time
+from types import SimpleNamespace
+
+
+def spawn_run(argv: list[str], input: str | None = None, timeout: float = 120.0):
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)